ktor-client/ktor-client-core/common/src/io/ktor/client/call/HttpClientCall.kt

```diff
@@ -158,7 +159,7 @@ class DoubleReceiveException(call: HttpClientCall) : IllegalStateException() {
  * Exception representing fail of the response pipeline
  * [cause] contains origin pipeline exception
  */
-@Suppress("KDocMissingDocumentation")
+@Suppress("KDocMissingDocumentation", "unused")
```

Comment:
Why `unused` is in uppercase somewhere and in lowercase here?

Reply:
Well.. we have a lot of both in the codebase. Not sure exactly why